index.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express")
+const app = express();
+require("dotenv").config();
+const authenticationRouter = require("./routes/authenticationRouter")
+const bodyParser = require("body-parser");
```

Comment:
all library improts should be on top then our js files should be imported. @ashok-choudhary-0